builtin/config.c

```diff
@@ -44,6 +44,7 @@ static struct config_options config_options;
 static int show_origin;
 static int show_scope;
 static int fixed_value;
+static char *comment;
```

Comment:
Let's use the type `const char *`, as we're not taking ownership of the memory (we're not `free()`ing it at the end; it's a command-line option that was not `malloc()`ed).